tests/dialects/test_gpu.py

```diff
@@ -253,6 +261,56 @@ def test_launch():
     assert nd_launch.dynamicSharedMemorySize is ten.result
 
 
+def test_launchfunc():
+    kernel = builtin.SymbolRefAttr("root", ["gpu", "kernel"])
+    args = [arith.Constant.from_int_and_width(10, builtin.IndexType())]
+    ten = arith.Constant.from_int_and_width(10, builtin.IndexType())
```

Comment:
tex -> x_size?

Reply:
y...z... etc

Reply:
Not sure I want that honestly.
It's meant as a unit test for the constructor rather than an example, so optimized for minimal overhead/unrelated maintenance.
Would you prefer making clearer that this value is not the point by using the test dialect?